doc/README.md

```diff
@@ -24,18 +24,18 @@ Unpack the files into a directory and run:
 
 ### Windows
 
-Unpack the files into a directory, and then run bitcoin-qt.exe.
+Unpack the files into a directory, and then run namecoin-qt.exe.
 
 ### OSX
 
-Drag Bitcoin-Qt to your applications folder, and then run Bitcoin-Qt.
+Drag Namecoin-Qt to your applications folder, and then run Namecoin-Qt.
 
 ### Need Help?
 
-* See the documentation at the [Bitcoin Wiki](https://en.bitcoin.it/wiki/Main_Page)
+* See the documentation at the [Namecoin Wiki](https://wiki.namecoin.org/index.php?title=Welcome)
 for help and more information.
-* Ask for help on [#bitcoin](http://webchat.freenode.net?channels=bitcoin) on Freenode. If you don't have an IRC client use [webchat here](http://webchat.freenode.net?channels=bitcoin).
-* Ask for help on the [BitcoinTalk](https://bitcointalk.org/) forums, in the [Technical Support board](https://bitcointalk.org/index.php?board=4.0).
+* Ask for help on [#namecoin-dev](http://webchat.freenode.net?channels=namecoin-dev) on Freenode. If you don't have an IRC client use [webchat here](http://webchat.freenode.net?channels=namecoin-dev).
```

Comment:
@josephbisch Listing both would probably be good.  #namecoin is for end user and general discussion, #namecoin-dev is for developer discussion.